packages/go_router/lib/src/information_provider.dart

```diff
@@ -111,9 +110,9 @@ class GoRouteInformationProvider extends RouteInformationProvider
 }
 
 /// A debug class that is used for asserting the [GoRouteInformationProvider] is
-/// in use with the [GoRouteInformationParser].
+/// in use with the [RouteInformationParser].
```

Comment:
this meant to specifically mention GoRouteInformationParser, can you revert this change?

Reply:
good catch - fixed